nautobot/core/apps/__init__.py

```diff
@@ -691,6 +691,66 @@ def convert_biginteger(field, registry=None):  # pylint: disable=redefined-outer
 
         monkey_mix(TaggableManager, mixins.TaggableManagerMonkeyMixin)
 
+        from django.db.models.sql.query import Query
+        from django.db.models.expressions import Ref
+
+        def set_values(self, fields):
+            self.select_related = False
+            self.clear_deferred_loading()
+            self.clear_select_fields()
+            self.selected = fields
```

Comment:
Note [here](https://github.com/django/django/blob/stable/4.2.x/django/db/models/sql/query.py#L2411-L2415) The fix is from 4.2  where `has_select_fields` is a no longer a class `property` which no longers require a setter to change its value. However, Django 3.2 in ltm still has
[`has_select_fields`](https://github.com/django/django/blob/stable/3.2.x/django/db/models/sql/query.py#L257-L259) as a class property which requires a setter method to change its value.
I wonder if we should just discard this line from the 4.2 fix, please double check me on that.